Automattic/jetpack
# Fix "Build artifacts" job and merge with "Update mirrors"

#### Changes proposed in this Pull Request:

When we removed .svnignore, that caused the job to start trying to
include all of node_modules and such in the artifact which is completely
not what we want.

We already have the logic to produce exactly what we do want for the
mirror repos, so merge those workflows (and separate building from pushing in the process).

#### Jetpack product discussion


p1610657658129800-slack-CBG1CP4EN

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* See that the new workflow built proper artifacts.
* See that the correct artifact was pushed to Jetpack Beta.

#### Proposed changelog entry for your changes:


* None needed.
